tools/maint/update_settings_docs.py

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/env python3
+# Copyright 2021 The Emscripten Authors.  All rights reserved.
+# Emscripten is available under two separate licenses, the MIT license and the
+# University of Illinois/NCSA Open Source License.  Both these licenses can be
+# found in the LICENSE file.
+
+"""Builds the emscripten website from source and creates a new commit & branch
+in the emscripten-site repository containing the changes."""
```

Comment:
This comment is out of date.

Can we replace it with a little bit of documentation on the script, e.g. a sample import or how it recognizes tags?